src/ui-test/tests/command.palette.test.ts

```diff
@@ -0,0 +1,57 @@
+import * as path from 'path';
+import {
+    after,
+    before,
+    BottomBarPanel,
+    EditorView,
+    SideBarView,
+    VSBrowser,
+    WebDriver,
+} from 'vscode-uitests-tooling';
+import {
+    CAMEL_ROUTE_YAML_WITH_SPACE,
+    CAMEL_RUN_ACTION_LABEL,
+    waitUntilTerminalHasText,
+    TEST_ARRAY_RUN,
+    executeCommand
+} from '../utils';
+
+describe('JBang commands execution through command palette', function () {
+    this.timeout(240000);
+
+    let editorView: EditorView;
+    let driver: WebDriver;
+
+    before('Before setup', async function () {
+        driver = VSBrowser.instance.driver;
+    });
+
+    after('After cleanup', async function () {
+        await new EditorView().closeAllEditors();
+    });
+
+    beforeEach('Before each test', async function () {
+        await VSBrowser.instance.openResources(path.resolve('src', 'ui-test', 'resources'));
+        await VSBrowser.instance.waitForWorkbench();
+
+        const section = await new SideBarView().getContent().getSection('resources');
+        await section?.openItem(CAMEL_ROUTE_YAML_WITH_SPACE);
+
+        editorView = new EditorView();
+        await section.getDriver().wait(async function () {
```

Comment:
in case you are using `driver = VSBrowser.instance.driver;` its better to use only this driver instance in tests.

means the line should looks like `await driver.wait(async function () {....`